meson.build

```diff
@@ -460,6 +460,7 @@ if build_tools
                                        install: true,
                                        install_dir: dir_libexec)
     install_man('tools/xkbcli-compile-keymap.1')
+    configh_data.set10('HAVE_XKBCLI_COMPILE_KEYMAP', true)
```

Comment:
This needs to be a separate commit, it seems unrelated to the compose tool here.